Source/WebKit/WebKitSwift/RealityKit/RKEntity.swift

```diff
@@ -347,13 +351,17 @@ public final class WKSRKEntity: NSObject {
         }
     }
     
-    @objc(applyDefaultIBL) func applyDefaultIBL()
+    @objc(applyDefaultIBLWithAttributionHandler:) func applyDefaultIBL(attributionHandler: @escaping (REAssetRef) -> Void)
     {
         Task {
             do {
 #if canImport(RealityFoundation, _version: 380)
                 let environment = await EnvironmentResource.defaultObject()
                 await MainActor.run {
+                    if let coreEnvironmentResourceAsset = environment.coreIBLAsset?.__as(REAssetRef.self) {
```

Comment:
is there a non underscore version of `__as` too that you could use?

Reply:
Unfortunately no.

Reply:
seeing as all this does is a forced cast anyways, maybe this can just be 

```suggestion
                    if let coreEnvironmentResourceAsset = environment.coreIBLAsset as? REAssetRef {
```

? I think this is better for multiple reasons:
1. It avoids using the weird `__as` function
2. It won't crash if it's not that type
3. It doesn't obfuscate what is actually happening (a cast)

Reply:
Unfortunately that casts differently from `__as` and ends up returning null.

Reply:
Checked with Richard offline and we'll stick with what we have for now.